pygmt/src/sphinterpolate.py

```diff
@@ -0,0 +1,69 @@
+"""
+sphinterpolate - Spherical gridding in tension of data on a sphere
+"""
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+from pygmt.io import load_dataarray
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    I="spacing",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(I="sequence", R="sequence")
+def sphinterpolate(data, **kwargs):
+    r"""
+    Create spherical grid files in tension of data.
+
+    Reads one or more ASCII [or binary] files (or standard input) containing
+    *lon, lat, z* and performs a Delaunay triangulation to set up a spherical
+    interpolation in tension. Several options may be used to affect the outcome, 
+    such as choosing local versus global gradient estimation or optimize the 
+    tension selection to satisfy one of four criteria.
```

Comment:
Style check was failing due to a line over 79 characters. Also, the 'standard input' part doesn't make sense in Python, so maybe simplify to just 'table'

```suggestion
    Reads a table containing *lon, lat, z* columns and performs a Delaunay
    triangulation to set up a spherical interpolation in tension. Several
    options may be used to affect the outcome, such as choosing local versus
    global gradient estimation or optimize the tension selection to satisfy one
    of four criteria.
```